### ISSUE TITLE
File history shows blank pages when sync conflict was resolved automatically

### ISSUE THREAD

**Issue by vibl** [siyuan-note/siyuan#8563]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Can the issue be reproduced with the default theme (daylight/midnight)?

- [X] I was able to reproduce the issue with the default theme

### Could the issue be due to extensions?

- [X] I've ruled out the possibility that the extension is causing the problem.

### Describe the problem

I've setup sync between two computers with an S3 storage, and it seems to work apart from the automatic conflict resolution feature:
- When modifying the same line in both computers with a different edit, one of the edit "wins" (which is fine), and the computer that has won has the edit of the other computer as a version in the file history.
- But the computer that has not won does not have its own edit in the file history. Instead, there are two blank (i.e. empty) versions of the file recorded in the file history.

### Expected result

After an automatic conflict resolution, each side should be able to retrieve the version that has "lost" the conflict resolution in the file history.

### Version environment

```markdown
- Version: v2.9.0
- Operating System: Kubuntu 23.04
- Browser (if used):
```

### Log file

I 2023/06/06 22:54:00 working.go:117: 
      ___                                   ___           ___           ___
     /  /\        ___           ___        /__/\         /  /\         /__/\
    /  /:/_      /  /\         /__/|       \  \:\       /  /::\        \  \:\
   /  /:/ /\    /  /:/        |  |:|        \  \:\     /  /:/\:\        \  \:\
  /  /:/ /::\  /__/::\        |  |:|    ___  \  \:\   /  /:/~/::\   _____\__\:\
 /__/:/ /:/\:\ \__\/\:\__   __|__|:|   /__/\  \__\:\ /__/:/ /:/\:\ /__/::::::::\
 \  \:\/:/~/:/    \  \:\/\ /__/::::\   \  \:\ /  /:/ \  \:\/:/__\/ \  \:\~~\~~\/
  \  \::/ /:/      \__\::/    ~\~~\:\   \  \:\  /:/   \  \::/       \  \:\  ~~~
   \__\/ /:/       /__/:/       \  \:\   \  \:\/:/     \  \:\        \  \:\
     /__/:/        \__\/         \__\/    \  \::/       \  \:\        \  \:\
     \__\/                                 \__\/         \__\/         \__\/
I 2023/06/06 22:54:00 runtime.go:70: kernel is booting:
    * ver [2.9.0]
    * arch [amd64]
    * os [ubuntu]
    * pid [298839]
    * runtime mode [prod]
    * working directory [/tmp/.mount_siyuank6kdvw/resources]
    * read only [false]
    * container [std]
    * database [ver=20220501]
    * workspace directory [/home/vianney/_/siyuan/vibl]
I 2023/06/06 22:54:00 conf.go:108: initialized the specified language [en_US]
I 2023/06/06 22:54:00 runtime.go:124: use network proxy [system]
I 2023/06/06 22:54:00 serve.go:113: kernel [pid=298839] http server [127.0.0.1:35997] is booting
I 2023/06/06 22:54:00 database.go:88: the database structure is changed, rebuilding database...
I 2023/06/06 22:54:00 database.go:108: reinitialized database [/home/vianney/_/siyuan/vibl/temp/siyuan.db]
I 2023/06/06 22:54:00 conf.go:690: database size [4.1 kB], tree/block count [0/0]
I 2023/06/06 22:54:00 working.go:155: kernel booted
I 2023/06/06 22:54:00 tesseract.go:153: tesseract-ocr enabled [ver=5.3.0, maxSize=2.0 MB, langs=eng+osd]
I 2023/06/06 22:54:00 box.go:79: auto stat [trees=0, blocks=0, dataSize=33 kB, assetsSize=0 B]
I 2023/06/06 22:54:00 disk.go:33: disk usage [total=502 GB, used=429 GB, free=48 GB]
I 2023/06/06 22:54:01 pandoc.go:138: initialized built-in pandoc [ver=, bin=/home/vianney/_/siyuan/vibl/temp/pandoc/bin/pandoc]
I 2023/06/06 22:54:01 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.01s], tree [count=56, size=1.1 MB]
I 2023/06/06 22:54:01 index.go:206: resolved refs [25] in [10ms]
I 2023/06/06 22:54:01 serve.go:126: reverse proxy server [127.0.0.1:6806] is booting
E 2023/06/06 23:02:44 pandoc.go:46: pandoc convert output []
I 2023/06/06 23:03:11 index.go:133: rebuilt database for notebook [20230606230311-n14cje9] in [0.00s], tree [count=0, size=0 B]
I 2023/06/06 23:03:11 index.go:206: resolved refs [26] in [23ms]
E 2023/06/06 23:03:20 pandoc.go:46: pandoc convert output []
E 2023/06/06 23:04:01 pandoc.go:46: pandoc convert output []
W 2023/06/06 23:04:15 flashcard.go:362: builtin deck not found
I 2023/06/07 11:30:07 box.go:79: auto stat [trees=60, blocks=3636, dataSize=4.3 MB, assetsSize=1.9 MB]
I 2023/06/07 11:30:07 disk.go:33: disk usage [total=502 GB, used=430 GB, free=47 GB]
I 2023/06/07 13:30:07 box.go:79: auto stat [trees=61, blocks=3656, dataSize=4.3 MB, assetsSize=1.9 MB]
I 2023/06/07 13:30:07 disk.go:33: disk usage [total=502 GB, used=430 GB, free=47 GB]
I 2023/06/07 15:30:07 box.go:79: auto stat [trees=61, blocks=3657, dataSize=4.3 MB, assetsSize=1.9 MB]
I 2023/06/07 15:30:07 disk.go:33: disk usage [total=502 GB, used=430 GB, free=46 GB]
I 2023/06/07 16:33:09 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.01s], tree [count=56, size=1.1 MB]
I 2023/06/07 16:33:09 index.go:206: resolved refs [26] in [11ms]
I 2023/06/07 17:30:07 box.go:79: auto stat [trees=59, blocks=3659, dataSize=5.2 MB, assetsSize=2.7 MB]
I 2023/06/07 17:30:07 disk.go:33: disk usage [total=502 GB, used=430 GB, free=46 GB]
I 2023/06/07 19:30:07 box.go:79: auto stat [trees=59, blocks=3659, dataSize=5.2 MB, assetsSize=2.7 MB]
I 2023/06/07 19:30:07 disk.go:33: disk usage [total=502 GB, used=430 GB, free=46 GB]
I 2023/06/07 21:30:07 box.go:79: auto stat [trees=59, blocks=3659, dataSize=5.2 MB, assetsSize=2.7 MB]
I 2023/06/07 21:30:07 disk.go:33: disk usage [total=502 GB, used=431 GB, free=46 GB]
I 2023/06/07 21:37:40 conf.go:465: exiting kernel [force=false, execInstallPkg=0]
I 2023/06/07 21:37:40 database.go:1134: closed database
I 2023/06/07 21:37:40 conf.go:837: cleared workspace temp
I 2023/06/07 21:37:40 conf.go:512: exited kernel
I 2023/06/16 12:55:47 working.go:117: 
      ___                                   ___           ___           ___
     /  /\        ___           ___        /__/\         /  /\         /__/\
    /  /:/_      /  /\         /__/|       \  \:\       /  /::\        \  \:\
   /  /:/ /\    /  /:/        |  |:|        \  \:\     /  /:/\:\        \  \:\
  /  /:/ /::\  /__/::\        |  |:|    ___  \  \:\   /  /:/~/::\   _____\__\:\
 /__/:/ /:/\:\ \__\/\:\__   __|__|:|   /__/\  \__\:\ /__/:/ /:/\:\ /__/::::::::\
 \  \:\/:/~/:/    \  \:\/\ /__/::::\   \  \:\ /  /:/ \  \:\/:/__\/ \  \:\~~\~~\/
  \  \::/ /:/      \__\::/    ~\~~\:\   \  \:\  /:/   \  \::/       \  \:\  ~~~
   \__\/ /:/       /__/:/       \  \:\   \  \:\/:/     \  \:\        \  \:\
     /__/:/        \__\/         \__\/    \  \::/       \  \:\        \  \:\
     \__\/                                 \__\/         \__\/         \__\/
I 2023/06/16 12:55:47 runtime.go:70: kernel is booting:
    * ver [2.9.0]
    * arch [amd64]
    * os [ubuntu]
    * pid [1857997]
    * runtime mode [prod]
    * working directory [/tmp/.mount_siyuanTChzRT/resources]
    * read only [false]
    * container [std]
    * database [ver=20220501]
    * workspace directory [/home/vianney/_/siyuan/vibl]
I 2023/06/16 12:55:47 runtime.go:124: use network proxy [system]
I 2023/06/16 12:55:47 serve.go:113: kernel [pid=1857997] http server [127.0.0.1:42113] is booting
I 2023/06/16 12:55:47 blocktree.go:448: read block tree [984 kB] to [/home/vianney/_/siyuan/vibl/temp/blocktree], elapsed [0.02s]
I 2023/06/16 12:55:47 conf.go:690: database size [10 MB], tree/block count [59/3658]
I 2023/06/16 12:55:47 tesseract.go:153: tesseract-ocr enabled [ver=5.3.0, maxSize=2.0 MB, langs=eng+osd]
I 2023/06/16 12:55:47 working.go:155: kernel booted
I 2023/06/16 12:55:48 box.go:79: auto stat [trees=3, blocks=1638, dataSize=4.8 MB, assetsSize=2.7 MB]
I 2023/06/16 12:55:48 disk.go:33: disk usage [total=502 GB, used=442 GB, free=35 GB]
I 2023/06/16 12:55:48 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.02s], tree [count=56, size=1.1 MB]
I 2023/06/16 12:55:48 serve.go:126: reverse proxy server [127.0.0.1:6806] is booting
I 2023/06/16 12:55:48 pandoc.go:138: initialized built-in pandoc [ver=, bin=/home/vianney/_/siyuan/vibl/temp/pandoc/bin/pandoc]
I 2023/06/16 12:56:18 process.go:36: received os signal [interrupt], exit kernel process now
I 2023/06/16 12:56:18 conf.go:465: exiting kernel [force=false, execInstallPkg=1]
I 2023/06/16 12:56:18 database.go:1134: closed database
I 2023/06/16 12:56:18 conf.go:837: cleared workspace temp
I 2023/06/16 12:56:49 working.go:117: 
      ___                                   ___           ___           ___
     /  /\        ___           ___        /__/\         /  /\         /__/\
    /  /:/_      /  /\         /__/|       \  \:\       /  /::\        \  \:\
   /  /:/ /\    /  /:/        |  |:|        \  \:\     /  /:/\:\        \  \:\
  /  /:/ /::\  /__/::\        |  |:|    ___  \  \:\   /  /:/~/::\   _____\__\:\
 /__/:/ /:/\:\ \__\/\:\__   __|__|:|   /__/\  \__\:\ /__/:/ /:/\:\ /__/::::::::\
 \  \:\/:/~/:/    \  \:\/\ /__/::::\   \  \:\ /  /:/ \  \:\/:/__\/ \  \:\~~\~~\/
  \  \::/ /:/      \__\::/    ~\~~\:\   \  \:\  /:/   \  \::/       \  \:\  ~~~
   \__\/ /:/       /__/:/       \  \:\   \  \:\/:/     \  \:\        \  \:\
     /__/:/        \__\/         \__\/    \  \::/       \  \:\        \  \:\
     \__\/                                 \__\/         \__\/         \__\/
I 2023/06/16 12:56:49 runtime.go:70: kernel is booting:
    * ver [2.9.0]
    * arch [amd64]
    * os [ubuntu]
    * pid [1858655]
    * runtime mode [prod]
    * working directory [/tmp/.mount_siyuanf0FLO5/resources]
    * read only [false]
    * container [std]
    * database [ver=20220501]
    * workspace directory [/home/vianney/_/siyuan/vibl]
I 2023/06/16 12:56:49 runtime.go:124: use network proxy [system]
I 2023/06/16 12:56:49 serve.go:113: kernel [pid=1858655] http server [127.0.0.1:38773] is booting
I 2023/06/16 12:56:49 tesseract.go:153: tesseract-ocr enabled [ver=5.3.0, maxSize=2.0 MB, langs=eng+osd]
I 2023/06/16 12:56:49 blocktree.go:448: read block tree [984 kB] to [/home/vianney/_/siyuan/vibl/temp/blocktree], elapsed [0.00s]
I 2023/06/16 12:56:49 conf.go:690: database size [11 MB], tree/block count [60/3656]
I 2023/06/16 12:56:49 working.go:155: kernel booted
I 2023/06/16 12:56:49 box.go:79: auto stat [trees=4, blocks=1640, dataSize=1.4 MB, assetsSize=4 B]
I 2023/06/16 12:56:49 disk.go:33: disk usage [total=502 GB, used=442 GB, free=35 GB]
I 2023/06/16 12:56:49 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.01s], tree [count=56, size=1.1 MB]
I 2023/06/16 12:56:50 pandoc.go:138: initialized built-in pandoc [ver=, bin=/home/vianney/_/siyuan/vibl/temp/pandoc/bin/pandoc]
I 2023/06/16 12:56:50 serve.go:126: reverse proxy server [127.0.0.1:6806] is booting
I 2023/06/16 13:03:27 index.go:133: rebuilt database for notebook [20230616130327-ed9crw2] in [0.00s], tree [count=0, size=0 B]
I 2023/06/16 13:03:27 index.go:206: resolved refs [26] in [21ms]
I 2023/06/16 13:45:25 repository.go:1170: synced data repo [provider=2, ufc=135, dfc=1, ucc=133, dcc=0, ub=1.5 MB, db=0 B] in [8.02s]
I 2023/06/16 13:46:17 repository.go:1170: synced data repo [provider=2, ufc=3, dfc=1, ucc=1, dcc=0, ub=6.7 kB, db=6.1 kB] in [0.48s]
I 2023/06/16 13:46:49 repository.go:1170: synced data repo [provider=2, ufc=0, dfc=1, ucc=0, dcc=0, ub=0 B, db=6.0 kB] in [0.22s]
I 2023/06/16 13:54:53 repository.go:1170: synced data repo [provider=2, ufc=4, dfc=24, ucc=1, dcc=22, ub=7.8 kB, db=273 kB] in [1.83s]
I 2023/06/16 13:54:59 repository.go:1170: synced data repo [provider=2, ufc=0, dfc=1, ucc=0, dcc=0, ub=0 B, db=6.9 kB] in [0.06s]
I 2023/06/16 13:57:52 repository.go:1170: synced data repo [provider=2, ufc=10, dfc=9, ucc=6, dcc=6, ub=7.1 kB, db=12 kB] in [2.00s]
I 2023/06/16 13:58:33 repository.go:1170: synced data repo [provider=2, ufc=3, dfc=1, ucc=1, dcc=0, ub=7.8 kB, db=7.3 kB] in [0.52s]
I 2023/06/16 13:58:56 repository.go:1170: synced data repo [provider=2, ufc=3, dfc=2, ucc=1, dcc=1, ub=4.7 kB, db=8.1 kB] in [0.49s]
I 2023/06/16 13:59:03 repository.go:1170: synced data repo [provider=2, ufc=0, dfc=1, ucc=0, dcc=0, ub=0 B, db=6.9 kB] in [0.09s]
I 2023/06/16 14:04:43 repository.go:1170: synced data repo [provider=2, ufc=3, dfc=2, ucc=1, dcc=1, ub=4.9 kB, db=9.1 kB] in [1.35s]
I 2023/06/16 14:05:35 conf.go:465: exiting kernel [force=false, execInstallPkg=0]
I 2023/06/16 14:05:35 sync.go:196: sync before exit
I 2023/06/16 14:05:35 repository.go:1170: synced data repo [provider=2, ufc=0, dfc=1, ucc=0, dcc=0, ub=0 B, db=6.9 kB] in [0.08s]
I 2023/06/16 14:05:35 database.go:1134: closed database
I 2023/06/16 14:05:35 conf.go:837: cleared workspace temp
I 2023/06/16 14:05:35 conf.go:512: exited kernel
I 2023/06/16 14:05:54 working.go:117: 
      ___                                   ___           ___           ___
     /  /\        ___           ___        /__/\         /  /\         /__/\
    /  /:/_      /  /\         /__/|       \  \:\       /  /::\        \  \:\
   /  /:/ /\    /  /:/        |  |:|        \  \:\     /  /:/\:\        \  \:\
  /  /:/ /::\  /__/::\        |  |:|    ___  \  \:\   /  /:/~/::\   _____\__\:\
 /__/:/ /:/\:\ \__\/\:\__   __|__|:|   /__/\  \__\:\ /__/:/ /:/\:\ /__/::::::::\
 \  \:\/:/~/:/    \  \:\/\ /__/::::\   \  \:\ /  /:/ \  \:\/:/__\/ \  \:\~~\~~\/
  \  \::/ /:/      \__\::/    ~\~~\:\   \  \:\  /:/   \  \::/       \  \:\  ~~~
   \__\/ /:/       /__/:/       \  \:\   \  \:\/:/     \  \:\        \  \:\
     /__/:/        \__\/         \__\/    \  \::/       \  \:\        \  \:\
     \__\/                                 \__\/         \__\/         \__\/
I 2023/06/16 14:05:54 runtime.go:70: kernel is booting:
    * ver [2.9.0]
    * arch [amd64]
    * os [ubuntu]
    * pid [1875767]
    * runtime mode [prod]
    * working directory [/tmp/.mount_siyuanppQI66/resources]
    * read only [false]
    * container [std]
    * database [ver=20220501]
    * workspace directory [/home/vianney/_/siyuan/vibl]
I 2023/06/16 14:05:54 runtime.go:124: use network proxy [system]
I 2023/06/16 14:05:54 serve.go:113: kernel [pid=1875767] http server [127.0.0.1:45829] is booting
I 2023/06/16 14:05:54 tesseract.go:153: tesseract-ocr enabled [ver=5.3.0, maxSize=2.0 MB, langs=eng+osd]
I 2023/06/16 14:05:55 pandoc.go:138: initialized built-in pandoc [ver=, bin=/home/vianney/_/siyuan/vibl/temp/pandoc/bin/pandoc]
I 2023/06/16 14:05:55 serve.go:126: reverse proxy server [127.0.0.1:6806] is booting
I 2023/06/16 14:05:56 sync.go:151: sync before boot
I 2023/06/16 14:05:56 repository.go:1069: boot get sync cloud files elapsed [0.23s]
I 2023/06/16 14:05:56 blocktree.go:448: read block tree [6.2 MB] to [/home/vianney/_/siyuan/vibl/temp/blocktree], elapsed [0.03s]
I 2023/06/16 14:05:56 conf.go:690: database size [77 MB], tree/block count [169/24468]
I 2023/06/16 14:05:56 working.go:155: kernel booted
I 2023/06/16 14:05:56 box.go:79: auto stat [trees=169, blocks=23920, dataSize=18 MB, assetsSize=1.9 MB]
I 2023/06/16 14:05:56 disk.go:33: disk usage [total=502 GB, used=442 GB, free=35 GB]
I 2023/06/16 14:05:56 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.01s], tree [count=56, size=1.1 MB]
I 2023/06/16 14:10:59 repository.go:1170: synced data repo [provider=2, ufc=5, dfc=1, ucc=1, dcc=0, ub=8.2 kB, db=7.4 kB] in [1.15s]
I 2023/06/16 14:19:01 repository.go:1170: synced data repo [provider=2, ufc=2, dfc=1, ucc=0, dcc=0, ub=6.9 kB, db=7.4 kB] in [0.89s]
I 2023/06/16 14:27:04 repository.go:1170: synced data repo [provider=2, ufc=2, dfc=1, ucc=0, dcc=0, ub=6.9 kB, db=7.4 kB] in [1.33s]
I 2023/06/16 14:35:07 repository.go:1170: synced data repo [provider=2, ufc=2, dfc=1, ucc=0, dcc=0, ub=6.9 kB, db=7.4 kB] in [1.01s]
I 2023/06/16 14:51:09 repository.go:1170: synced data repo [provider=2, ufc=2, dfc=1, ucc=0, dcc=0, ub=6.9 kB, db=7.4 kB] in [1.18s]
I 2023/06/16 15:18:52 conf.go:465: exiting kernel [force=false, execInstallPkg=0]
I 2023/06/16 15:18:52 sync.go:196: sync before exit
I 2023/06/16 15:18:53 repository.go:1170: synced data repo [provider=2, ufc=2, dfc=1, ucc=0, dcc=0, ub=6.9 kB, db=7.4 kB] in [1.01s]
I 2023/06/16 15:18:53 database.go:1134: closed database
I 2023/06/16 15:18:53 conf.go:837: cleared workspace temp
I 2023/06/16 15:18:54 conf.go:512: exited kernel
I 2023/06/16 16:48:23 working.go:117: 
      ___                                   ___           ___           ___
     /  /\        ___           ___        /__/\         /  /\         /__/\
    /  /:/_      /  /\         /__/|       \  \:\       /  /::\        \  \:\
   /  /:/ /\    /  /:/        |  |:|        \  \:\     /  /:/\:\        \  \:\
  /  /:/ /::\  /__/::\        |  |:|    ___  \  \:\   /  /:/~/::\   _____\__\:\
 /__/:/ /:/\:\ \__\/\:\__   __|__|:|   /__/\  \__\:\ /__/:/ /:/\:\ /__/::::::::\
 \  \:\/:/~/:/    \  \:\/\ /__/::::\   \  \:\ /  /:/ \  \:\/:/__\/ \  \:\~~\~~\/
  \  \::/ /:/      \__\::/    ~\~~\:\   \  \:\  /:/   \  \::/       \  \:\  ~~~
   \__\/ /:/       /__/:/       \  \:\   \  \:\/:/     \  \:\        \  \:\
     /__/:/        \__\/         \__\/    \  \::/       \  \:\        \  \:\
     \__\/                                 \__\/         \__\/         \__\/
I 2023/06/16 16:48:23 runtime.go:70: kernel is booting:
    * ver [2.9.0]
    * arch [amd64]
    * os [ubuntu]
    * pid [1894066]
    * runtime mode [prod]
    * working directory [/tmp/.mount_siyuanJP6pjk/resources]
    * read only [false]
    * container [std]
    * database [ver=20220501]
    * workspace directory [/home/vianney/_/siyuan/vibl]
I 2023/06/16 16:48:23 runtime.go:124: use network proxy [system]
I 2023/06/16 16:48:23 serve.go:113: kernel [pid=1894066] http server [127.0.0.1:46659] is booting
I 2023/06/16 16:48:23 tesseract.go:153: tesseract-ocr enabled [ver=5.3.0, maxSize=2.0 MB, langs=eng+osd]
I 2023/06/16 16:48:23 pandoc.go:138: initialized built-in pandoc [ver=, bin=/home/vianney/_/siyuan/vibl/temp/pandoc/bin/pandoc]
I 2023/06/16 16:48:24 serve.go:126: reverse proxy server [127.0.0.1:6806] is booting
I 2023/06/16 16:48:24 sync.go:151: sync before boot
I 2023/06/16 16:48:25 repository.go:1069: boot get sync cloud files elapsed [1.14s]
I 2023/06/16 16:48:25 blocktree.go:448: read block tree [6.2 MB] to [/home/vianney/_/siyuan/vibl/temp/blocktree], elapsed [0.02s]
I 2023/06/16 16:48:25 conf.go:690: database size [77 MB], tree/block count [169/24468]
I 2023/06/16 16:48:25 working.go:155: kernel booted
I 2023/06/16 16:48:25 box.go:79: auto stat [trees=169, blocks=24468, dataSize=18 MB, assetsSize=1.9 MB]
I 2023/06/16 16:48:25 disk.go:33: disk usage [total=502 GB, used=442 GB, free=35 GB]
I 2023/06/16 16:48:25 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.01s], tree [count=56, size=1.1 MB]
I 2023/06/16 16:53:28 repository.go:1170: synced data repo [provider=2, ufc=5, dfc=1, ucc=0, dcc=0, ub=8.2 kB, db=7.4 kB] in [1.46s]
I 2023/06/16 17:01:11 repository.go:1170: synced data repo [provider=2, ufc=0, dfc=1, ucc=0, dcc=0, ub=0 B, db=6.9 kB] in [0.76s]
I 2023/06/16 17:01:30 repository.go:1170: synced data repo [provider=2, ufc=0, dfc=1, ucc=0, dcc=0, ub=0 B, db=6.9 kB] in [0.06s]
I 2023/06/16 17:09:32 repository.go:1170: synced data repo [provider=2, ufc=0, dfc=1, ucc=0, dcc=0, ub=0 B, db=6.9 kB] in [0.76s]

**Comment by vibl** [comment]
Same exact problem in Data History.

**Comment by vibl** [comment]
By the way, thanks so much for Siyuan, which seems to be an amazing piece of software! I'm a long-time user of many note-taking apps and outliners and it seems much better than any other I've tried so far. I'm considering migrating to it from Logseq.

**Comment by 88250** [comment]
There is a problem that the input data is not saved in v2.9.0, please upgrade to v2.9.1 first.

Thank you for your attention and trial to SiYuan. We have closed the issue. If you have any questions, please feel free to post.

**Comment by vibl** [comment]
Confirmed: this is solved in v2.9.1
